invent-code-0/main.cpp

```diff
@@ -6,12 +6,14 @@ int main() {
   int price_avocado;
   int price_tomato;
   int num_avocados;
-  int num_tomatoes;
-  // Step 1: Prompt the user to input the number of tomatoes and avocados required.
+  int num_tomatoes,total_cost;
 
+  // Step 1: Prompt the user to input the number of tomatoes and avocados required.
+    cin >> num_avocados >> num_tomatoes;
```

Comment:
with cin, you should alwyas prompt the user. Or how will they know what they are entering?